apps/classifications/rules.py

```diff
@@ -1,9 +1,16 @@
 import rules
 
+from adhocracy4.modules import predicates as module_predicates
 from adhocracy4.projects.predicates import is_moderator as is_project_moderator
 
 rules.add_perm('a4_candy_classifications.view_userclassification',
                is_project_moderator)
 
 rules.add_perm('a4_candy_classifications.view_aiclassification',
                is_project_moderator)
+
+rules.add_perm('a4_candy_classifications.change_userclassification',
+               module_predicates.is_context_moderator)
```

Comment:
using this rule produced
```
{"detail":"You do not have permission to perform this action."}
```

while using the other one (`is_project_moderator`) gave a real long error (try it!) :)

Reply:
So I actually think you got it all right, but the token authentication thing from Aplus got in the way.. I didnt find out how to circumvent that with curl, but if you call /api/login first with the normal credentials, you get a token and can use that with curl...

We need to use the module_predicates.is_context_moderator here, because for the patch request, the permission_object is the classification (that happens in ViewSetRulesPermission again :)) and not the project..